runtime/lua/vim/filetype/detect.lua

```diff
@@ -1322,6 +1322,24 @@ function M.txt(bufnr)
   end
 end
 
+-- Determine if a .v file is Verilog, V, or Coq
+function M.v(bufnr)
+  if vim.fn.did_filetype() ~= 0 then
+    -- Filetype was already detected
+    return
+  end
+  for _, line in ipairs(getlines(bufnr, 1, 200)) do
+    if not line:find('^%s*/') then
+      if line:find(';%s*/.*$') then
```

Comment:
This doesn't look right, the vimpatch looks for an optional comment.

Reply:
Yes, that's what I meant, you can't have optional captures in Lua, IIUC.